### DISCUSSION TITLE
where is the resource file

### DISCUSSION THREAD

**Discussion by oakmill** [raysan5/raylib#1908] [General]
i  am new in  raylib, when i tried to run  examples from raylib webpage,
i noticed it need resource file in order to compile.  for example
(LoadTexture("resources/cyberpunk_street_background.png"))
my question is where can i locate those resource files.
thank you for your attention
oakmill

**Comment by raysan5** [comment]
Hi @oakmill, resource files should be next to the compiled executable. For example, raylib examples also include the resources directory in the same directory as the source code but it depends on every project organization. Sometimes resources/assets are placed in a separate folder, just in the directory where the executable will be generated on compilation. 

In the case of that specific asset, it can be found in raylib textures examples: https://github.com/raysan5/raylib/tree/master/examples/textures/resources

**Comment by oakmill** [comment]
hi ray,
thank you, i got it.
oakmill

Ray ***@***.***> 於 2021年8月8日 週日 下午12:34寫道：
